viz/components/dust/slideshow/v1/index.tsx

```diff
@@ -188,7 +188,7 @@ export const Columns = ({
 
   const hasTitle = Boolean(title);
   const hasDesc = Boolean(description);
-  const startRow = 1 + (hasTitle ? 1 : 0) + (hasDesc ? 1 : 0); // Where cards begin.
+  const startRow = 1 + (hasTitle ? 1 : 0); // Where cards begin.
```

Comment:
Description looks more natural together with Heading so I moved inside the same col